index.html

```diff
@@ -0,0 +1,133 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap-icons@1.8.1/font/bootstrap-icons.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Inter:wght@400;500&family=Poppins&family=Roboto:wght@700&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="index.css">
+    <title>Traveler Rentals  Cars</title>
+</head>
+
+
+    <header>
+       
+        <div class="navbar-container">
+            <img src="./pictures/facebook.png" alt="facebook">
+            <img src="./pictures/twitter.png" alt="twitter">
+            <ul class="navbarlog_container">
+                <li>English</li>
+                <li>My Page</li>
+                <li>Logout</li>
+            </ul>
+        </div>
+        
+      
+        <div class="topnav">
+            <!--hamburhuer tap-->
+              <nav>
+                <img  src="./pictures/hamburguer.png" class="hamburguer" alt="hamburguer">
+            </nav>
+
+            <h4  class="active" >Travelers Car <br>Rentals</h4>
+
+            <div id="Links">   
+                <ul>
+             <li ><img src="./pictures/close.png" class="close" alt="Close menu"></li>  
+             <li>  <a href="about.html" class="test">About</a></li> 
+             <li>  <a href="#" class="test">Program</a></li> 
+             <li>  <a href="#" class="test">Join</a></li> 
+             <li>  <a href="#" class="test">Sponsor</a></li> 
+             <li>  <a href="index.html" class="rent">Rent a Car</a></li> 
+            </ul>
+            </div>   
+            </div>      
+            <!--hamburguer tap-->    
+    </header>
+    <section>
+        <div class="main">
+            <h1>"Hello, Sharing world!"</h1>
+            <p class="creative">Travelers Rental cars.<br> 2020</p>
+            <div class="joyfull">
+                <p>A joyfull celebration believing in the value of opennes and sharing. creating a positive change with people from over 80 countrys is taking place in October. in USA</p>
+                <p class="dates">2022.04.14(THU)</p>
+                <p class="national">@ National  museum of USA. Art Center</p>
+            </div>
```

Comment:
please the `font` should match the design of the template. remember that this project is to help you perform your skills in front of pages

Reply:
change font got it going to change it thank you very much for your CR